app/views/capital_gains.scala.html

```diff
@@ -36,12 +36,10 @@
 @includes.taxs_main(title = Messages("ats.capital_gains_tax.html.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo), viewModel.utr, contentHeader = Option(headerContent), actingAttorney = actingAsAttorney) {
     @helpers.back_link(Messages("generic.back"))
     <h1>
-        <span class="gray heading-medium block-heading">
-          @Html(Messages("generic.tax_year_heading", viewModel.taxYearFrom, viewModel.taxYearTo))
-        </span>
-      @Html(Messages("ats.capital_gains_tax.subtitle"))
+      @Html(Messages("ats.capital_gains_tax.subtitle"))<br>
```

Comment:
Is this appropriate use of `br`?

Reply:
Thats one for @russellthorn

Reply:
In short, yes this is what `<br>`'s were designed for. But the long answer is - a span with a class would be more ideal for several reasons. We have a pattern for something similar, [headings with a caption](https://design-system.service.gov.uk/styles/typography/#headings-with-captions). But not a pattern for split level headings without styling like these. So I would have to add a local class with `display: block` to replace these `<br>`'s. Either way using `<br>` or `<span>` VoiceOver reads out the heading as 2 parts i.e. "Heading level 1, 2 parts...." then reads out the heading, then the 2 parts again. Much of a muchness.

Summary - `<br>` are fine and were a quick solution in a crazy world of silly deadlines and changing goal posts. But a span would be slightly more ideal but would need implementing on a local repo scale because there are no pre-existing patterns or classes for doing just this.